playbook/app/pb_kits/playbook/pb_advanced_table/_advanced_table.tsx

```diff
@@ -49,6 +58,9 @@ type AdvancedTableProps = {
   toggleExpansionIcon?: string | string[]
   onRowSelectionChange?: (arg: RowSelectionState) => void
   virtualizedRows?: boolean
+  fullscreenable?: boolean
```

Comment:
I see why you chose this prop name, makes sense meaning wise, but lets call it just `fullScreen`, its more in keeping with our naming conventions!